practice/pig-latin.py

```diff
@@ -1,58 +1,180 @@
+#     """Checks for punctuation at end of word and flips a boolean switch if it is present """
+#     return word[-1] in PUNCTUATIONS
+#
+# def save_punctuation(english_word):
+#     """saves the punctuation of the english word"""
+#     punctuation = english_word[-1]
+#     return punctuation
+#
+# def strip_punctuation(english_word):
+#     """cuts punctuation of english word"""
+#     return english_word[0:-1]
+#
+# def convert_english_word_to_pig_latin_word(english_word):
+#     """Translates english_word into pig_latin_word considering for up to 4 consonants in a row."""
+#     if english_word[0] in FIRST_VOWELS:
+#         pig_latin_word = english_word + 'yay'
+#     elif english_word in SPECIAL_CASES:
+#         pig_latin_word = english_word + 'yay'
+#     elif english_word[1] in LATER_VOWELS:
+#         pig_latin_word = english_word[1:] + english_word[:1] + 'ay'
+#     elif english_word[2] in LATER_VOWELS:
+#         pig_latin_word = english_word[2:] + english_word[:2] + 'ay'
+#     else:
+#         pig_latin_word = english_word[3:] + english_word[:3] + 'ay'
+#     return pig_latin_word
+#
+# def capitalize_pig_latin_word(temp_pig_latin_word):
+#     """If the english word was capitalized, this function appropriately capitalized the Pig Latin Word"""
+#     lowercase_first_letter = temp_pig_latin_word[0]
+#     uppercase_first_letter = lowercase_first_letter.capitalize()
+#     pig_latin_word = temp_pig_latin_word.replace(lowercase_first_letter, uppercase_first_letter, 1)
+#     return pig_latin_word
+#
+# def paste_punctution_back_on_pig_latin_word(temp_pig_latin_word, punctuation):
+#     """Adds punctuation to pig_latin_word as well as adding a space for a nice final phrase."""
+#     return temp_pig_latin_word + punctuation
+#
+# def print_full_pig_latin_phrase (phrase):
+#     print(phrase)
+#
+#
+# # 3. Main
+#
+#
+# def main():
+#     """Main Function"""
+#     pig_latin_phrase = ''
+#     english_phrase = get_phrase_casted_into_list()
+#
+#
+#
+#
+#     for english_word in english_phrase:
+#         capitalize_boolean = flip_capital_boolean(english_word)
+#         if capitalize_boolean == True:
+#             english_word = check_for_and_cut_capitals(english_word)
+#         punctuation_boolean = flip_punctuation_boolean(english_word)
+#         if punctuation_boolean == True:
+#             saved_punctuation = save_punctuation(english_word)
+#             english_word = strip_punctuation(english_word)
+#         temp_pig_latin_word = convert_english_word_to_pig_latin_word(english_word)
+#         if capitalize_boolean == True:
+#             temp_pig_latin_word = capitalize_pig_latin_word(temp_pig_latin_word)
+#         if punctuation_boolean == True:
+#             temp_pig_latin_word = paste_punctution_back_on_pig_latin_word (temp_pig_latin_word,saved_punctuation)
+#         pig_latin_phrase += temp_pig_latin_word + ' '
+#     print_full_pig_latin_phrase(pig_latin_phrase)
+# main()
+
+
+
+
+def main ():
```

Comment:
Looks like this was an in-progress version of the code. Find me and we can find if there are any commits that are not in this branch that contain the correct content.